.github/workflows/ci.yml

```
@@ -145,24 +145,16 @@ jobs:
-          path: .next
-          key: ${{ runner.os }}-nextjs-${{ hashFiles('package-lock.json') }}-${{ hashFiles('**/*.ts', '**/*.tsx', '**/*.js', '**/*.jsx') }}
-          restore-keys: |
-            ${{ runner.os }}-nextjs-${{ hashFiles('package-lock.json') }}-
-
-  build-check:
-    name: Build Check (Production)
-    # REQUIRED: Must use self-hosted runner exclusively
-    # DO NOT change to ubuntu-latest, windows-latest, or any GitHub-hosted runner
+  playwright-tests:
+    name: Playwright E2E Tests
+    # REQUIRED: Must use self-hosted runner exclusively  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)